app/src/main/java/org/simple/clinic/scheduleappointment/PotentialAppointmentDateModel.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.scheduleappointment
```

Comment:
Sorry I think I made a mistake, the comment was for a different commit 2ee2e229dafad4e54cb14e09b113225c1ae08d40. I accidentally left it for the first commit, can you revert this rename and do this for that commit, please 😅